### PR TITLE
4Pm-4: Update README.md with Heroku Production links 

### LINKED ISSUE #1: starter code heroku deployment and readme changes

### LINKED ISSUE #6: OAuth credentials (heroku, localhost, QA deployments)

### PR DESCRIPTION
**Pull request by vmreed1**
# Overview

Updated README.md with 4pm-4 Heroku link 

Closes #1, Closes #6

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,12 +8,12 @@
 * [QA](https://ucsb-cs156-w22.github.io/team04-w22-6pm-HappyCows-docs-qa/)
 
 # Heroku
-6pm-3:
-* [Production](https://team04-w22-6pm-3.herokuapp.com/)
-* [QA](https://team04-w22-6pm-3-qa.herokuapp.com/)
+4pm-3:
+* [Production](https://s22-4pm-happycows.herokuapp.com)
+* [QA](https://team04-w22-6pm-4-qa.herokuapp.com/)
```
Comment (pconrad) [inline README.md:13]: Should this link to s22-6pm-3?

File: README.md [hunk 2]
```diff
@@ -8,12 +8,12 @@
...
 
-6pm-4:
-* [Production](https://team04-w22-6pm-4.herokuapp.com/)
+4pm-4:
+* [Production](https://s22-4pm-happycows.herokuapp.com)
 * [QA](https://team04-w22-6pm-4-qa.herokuapp.com/)
```
Comment (pconrad) [inline README.md:17]: w22 or s22?

**Review by pconrad — CHANGES_REQUESTED** [review]
Check those links... I wonder if they are pointing where you think they are pointing?

File: README.md (hunk 1 above)
Comment (htn5) [reply README.md:13]: i believe the production link is correct. the qa link will be fixed in a later readme pull request update

**Review by WangHaoranRobin — APPROVED** [review]
lgtm

**Review by pconrad — DISMISSED** [review]

**Review by pconrad — COMMENTED** [review]
LGTM!

**Review by pconrad — APPROVED** [review]
LGTM

**Comment by pconrad** [comment]
10 pts because includes OAuth

**Pull request merged**